krummstab/__init__.py

```diff
@@ -365,6 +367,197 @@ def unsupported_marking_mode_error(_the_config: config.Config) -> None:
     logging.critical(f"Unsupported marking mode {_the_config.marking_mode}!")
 
 
+# ============================== Summarize Sub-Command =========================
+
+
+def summarize_points_per_exercise(_the_config: config.Config) -> None:
+    email_to_name = {}
+    for team in _the_config.teams:
+        for first_name, last_name, email in team:
+            email_to_name[email] = (first_name, last_name)
+
+    workbook = xlsxwriter.Workbook("Points_Summary_Report.xlsx")
+    worksheet = workbook.add_worksheet("Points Summary")
+
+    marks_files = args.marks_dir.glob("points_*_individual.json")
+    students_marks = {}
+    graded_sheet_names = {}
+
+    for file in marks_files:
+        with open(file, 'r') as f:
+            data = json.load(f)
+            sheet_name = data["adam_sheet_name"]
+            marks = data["marks"]
+            if sheet_name not in graded_sheet_names:
+                graded_sheet_names[sheet_name] = []
+
+            for email, exercises in marks.items():
+                if email not in students_marks:
+                    students_marks[email] = {}
+                if sheet_name not in students_marks[email]:
+                    students_marks[email][sheet_name] = {}
```

Comment:
This should produce a warning if two tutors marked the same exercise for one student.